app/assets/stylesheets/application.css.scss

```diff
@@ -43,6 +43,8 @@ header.navigation {
   $navigation-submenu-width: 12em;
   $horizontal-bar-mode: $large-screen;
 
+           line_that_hound_should_definitely_comment_on: 101 012 41024 010 241024 0125 1053 13205325 1;
```

Comment:
Line should be indented 2 spaces, but was indented 11 spaces<br>Properties should be ordered background-color, border-bottom, font-family, line_that_hound_should_definitely_comment_on, min-height, position, top, width, z-index<br>Unknown property line_that_hound_should_definitely_comment_on